music/src/core/player.ts

```diff
@@ -409,29 +399,35 @@ class DrumKit {
 export class Player extends BasePlayer {
   private drumKit = DrumKit.getInstance();
 
-  private bassSynth =
-      new Tone.Synth({volume: 5, oscillator: {type: 'triangle'}}).toMaster();
+  private bassSynth = new Tone.Synth({
+    volume: 5,
+    oscillator: { type: 'triangle' },
+  }).toDestination();
 
-  private polySynth = new Tone.PolySynth(10).toMaster();
+  private polySynth = new Tone.PolySynth().toDestination();
 
   /**
    * The Tone module being used.
    */
-  static readonly tone = Tone;  // tslint:disable-line:no-any
+  static readonly tone = Tone; // tslint:disable-line:no-any
 
   protected playNote(time: number, note: NoteSequence.INote) {
     // If there's a velocity, use it.
-    const velocity = note.hasOwnProperty('velocity') ?
-        note.velocity / constants.MAX_MIDI_VELOCITY :
-        undefined;
+    const velocity = note.hasOwnProperty('velocity')
+      ? note.velocity / constants.MAX_MIDI_VELOCITY
+      : undefined;
 
     if (note.isDrum) {
       this.drumKit.playNote(note.pitch, time, velocity);
     } else {
-      const freq = new Tone.Frequency(note.pitch, 'midi');
+      const freq = Tone.Frequency(note.pitch, 'midi').toFrequency();
```

Comment:
Why is the 'new' not needed anymore?

Reply:
because this is now a method and not a class